modules/Bio/EnsEMBL/Compara/PipeConfig/CreateSpeciesTree_conf.pm

```diff
@@ -280,17 +279,13 @@ sub pipeline_analyses {
               1 => ['copy_files_to_sketch_dir'],
           },
         },
-
+            
         { -logic_name => 'copy_files_to_sketch_dir',
-          -module     => 'Bio::EnsEMBL::Compara::RunnableDB::SpeciesTree::CopyFilesAsUser',
+          -module     => 'Bio::EnsEMBL::Hive::RunnableDB::SystemCmd',
           -parameters => {
               'destination_dir' => $self->o('sketch_dir'),
-              'become_user'     => $self->o('write_access_user'),
               'source_dir'      => $self->o('output_dir'),
-              'file_list' => [
-                '#source_dir#/*.msh',
-                '#source_dir#/#collection#',
-              ],
+              'cmd'             => 'cp #source_dir#/*.msh #destination_dir# && cp #source_dir#/#collection# #destination_dir#',
```

Comment:
```suggestion
              'cmd' => 'cp #source_dir#/*.msh #destination_dir# && cp #source_dir#/#collection# #destination_dir#',
```

Reply:
This is to align with the previous two parameters, so I'm going to leave it if you don't mind :)